### PR TITLE
Update Hongkong OCD-IDs as per changes in 2021.

### PR DESCRIPTION
**Pull request by novitam**
There are updates to HongKong's constituencies:

Abolished:
- Hong Kong Island is abolished [link](https://en.wikipedia.org/wiki/Hong_Kong_Island_(1998_constituency))
- New Territories West is abolished [link](https://en.wikipedia.org/wiki/New_Territories_West_(1998_constituency))
- New Territories East is abolished [link](https://en.wikipedia.org/wiki/New_Territories_East_(1998_constituency))

New list for Hong Kong geographical constituencies: [here](https://en.wikipedia.org/wiki/List_of_constituencies_of_Hong_Kong#2021%E2%80%93present)

Official press release for the update: https://www.info.gov.hk/gia/general/202106/01/P2021060100370.htm

### CODE REVIEW

**Comment by novitam** [comment]
@jloutsenhizer PTAL, thanks!

**Comment by novitam** [comment]
Friendly ping, @jloutsenhizer PTAL, thanks!

File: identifiers/country-hk/geographical_constituencies.csv [hunk 1]
```diff
@@ -1,6 +1,14 @@
-id,name
-ocd-division/country:hk/geographical_constituency:hk_island,Hong Kong Island
-ocd-division/country:hk/geographical_constituency:kowloon_west,Kowloon West
-ocd-division/country:hk/geographical_constituency:kowloon_east,Kowloon East
-ocd-division/country:hk/geographical_constituency:new_territories_west,New Territories West
-ocd-division/country:hk/geographical_constituency:new_territories_east,New Territories East
\ No newline at end of file
+id,name,validFrom,validTo
+ocd-division/country:hk/geographical_constituency:hk_island,Hong Kong Island,,2021-06-01
```
Comment (jloutsenhizer) [inline identifiers/country-hk/geographical_constituencies.csv:2]: The only thing unclear to me is whether 2021-06-01 is the correct date for the validTo/From entries. I see the map was published 2021-06-01 for public viewing, but are these districts changed immediately at that point or not until the 2021 election in December: https://en.wikipedia.org/wiki/2021_Hong_Kong_legislative_election

Comment (novitam) [reply identifiers/country-hk/geographical_constituencies.csv]: Good callout. The new constituencies apply from the next election. Changed the validTo/From dates to the date of the next election, which is 2021-12-19.

**Comment by novitam** [comment]
hello, PTAL @jloutsenhizer

**Review by jloutsenhizer — APPROVED** [review]

**Comment by novitam** [comment]
@jpmckinney please review, thanks!

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**